pkgs/development/python-modules/drf-extra-fields/default.nix

```
@@ -0,0 +1,39 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, setuptools
+, django
+, djangorestframework
+, filetype
+}:
+
+buildPythonPackage rec {
+  pname = "drf-extra-fields";
+  version = "3.7.0";
+  format = "pyproject";
```

Dunno why `format` doesn't generate a warning for new derivations yet, but we should all use `pyproject = true;` now
```suggestion
  pyproject = true;
```